v2_jobs.go

```diff
@@ -80,12 +83,17 @@ func (s *JobStatus) UnmarshalJSON(b []byte) error {
 }
 
 type Task struct {
-	ID       string `json:"uuid"`
-	ExitCode uint8  `json:"exit_code"`
+	ID          string `json:"uuid"`
+	ExitCode    uint8  `json:"exit_code"`
+	Filename    string `json:"file_name"`
+	CreatedAt   string `json:"time_created"`
+	StartedAt   string `json:"time_started"`
+	CompletedAt string `json:"time_ended"`
+	Duration    string `json:"duration"`
 }
 
 func (s *JobsServiceOp) List(ctx context.Context, ID string, r *JobsListRequest) ([]Job, *Response, error) {
-	path := fmt.Sprintf("%s/%s", jobsBasePath, ID)
+	path := fmt.Sprintf("%s/%s%s", jobsBasePath, ID, jobsDetailsQuery)
```

Comment:
This could or should be optional, e.g. a boolean attribute that is part of the request struct.

Use url.Values to encode query parameters. Here's a helper that could be used as a reference: https://github.com/digitalocean/godo/blob/c154e94a159362efd5f5b718cb49df029937ebd1/godo.go#L203.

Reply:
I'll check this out.